tests/test_get_ps_plus_1.py

```diff
@@ -0,0 +1,76 @@
+import pytest
```

Comment:
Would you mind to add an IDN test case here as well? E.g., https://github.com/mozilla/domain_utils/issues/5#issuecomment-546157822